packages/cicero-core/test/data/helloworldstate/README.md

```diff
@@ -0,0 +1,3 @@
+# Hello World State
+
+This is the stateful Hello World of Accord Project Templates. Executing the clause will simply echo back the text that occurs after the string `Hello` prepended to text that is passed in the request along with the number of times the clause has been called.
```

Comment:
Can you add a comment that includes the passphrase for the sample p12 keystore in this template, please?